modules/graph/vertex_map/arrow_vertex_map.h

```diff
@@ -303,8 +296,6 @@ class ArrowVertexMap
   std::vector<std::vector<vineyard::Hashmap<oid_t, vid_t>>> o2g_;
 
   friend class ArrowVertexMapBuilder<OID_T, VID_T>;
-
-  friend class gs::ArrowProjectedVertexMap<OID_T, VID_T>;
 };
```

Comment:
Remove the friend class declaration may break graphscope. Please double check the usages.

Reply:
I refine the `ProjectedVetexMap` in https://github.com/alibaba/GraphScope/pull/1794 and it's no need to access vertex_map's private member, so we can remove the friend declaration here.